04.bowling/bowling.rb

```diff
@@ -29,28 +29,30 @@ def get_frames(shots)
 end
 
 def calc_total_point(frames)
-  point = 0
-  (0..8).each do |i|
+  point = (0..8).sum do |i|
     current_frame = frames[i]
     next_frame = frames[i + 1]
     after_next_frame = frames[i + 2]
     is_one_before_final_frame = (i == 8)
 
-    point += calc_frame_point(current_frame, next_frame, after_next_frame, is_one_before_final_frame)
+    if spare?(current_frame)
+      10 + next_frame[0]
+    elsif strike?(current_frame) && !strike?(next_frame)
+      10 + next_frame[0] + next_frame[1]
+    elsif strike?(current_frame) && strike?(next_frame)
+      if is_one_before_final_frame
+        10 + next_frame[0] + next_frame[1]
+      else
+        10 + next_frame[0] + after_next_frame[0]
+      end
```

Comment:
こんな感じでストライクのときの計算処理を別メソッドに切り出すといいかもしれません。

```suggestion
    elsif strike?(current_frame)
      calc_strike_bonus(frames, i)
```

Reply:
思いつきませんでした・・読みやすくなりました、ありがとうございます！修正しました。https://github.com/mayumonj/ruby-practices/pull/3/commits/fa324651fae518af6673805aac101b6c2cd8db32